.github/workflows/bake-publish.yaml

```diff
@@ -107,27 +122,28 @@ jobs:
           echo "VERSION=${TAG,,}" >> ${GITHUB_ENV}
           echo "DOCKER_REGISTRY=${REG,,}" >> ${GITHUB_ENV}
           echo "DOCKER_ORG=${ORG,,}" >> ${GITHUB_ENV}
+          echo "ARTIFACT=${{ env.ARTIFACT_PATH }}/${{ inputs.artifact }}" >> ${GITHUB_ENV}
 
-      # 2.3 - Login against the docker registry
+      # 2.4 - Login against the docker registry
       - name: Login to registry ${{ inputs.registry }}
         uses: docker/login-action@v2
         with:
           registry: ${{ inputs.registry }}
           username: ${{ secrets.username }}
           password: ${{ secrets.password }}
 
-      # 2.4 - Setup QEMU for platform emulation
+      # 2.5 - Setup QEMU for platform emulation
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v2
 
-      # 2.5 - Setup Docker BuildX for multi platform building
+      # 2.6 - Setup Docker BuildX for multi platform building
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
         with:
```

Comment:
please remove this as well, or make a separate PR for this, it should no longer be needed.
It was pinned to a version due to an upgrade that was done earlier this year that caused issues, I believe this has been resolved now as I am not using it in any other version except in ASKEM

Reply:
comment seems off...
I mean the `with:` section that sets the driver version

Reply:
Can be done as a separate PR. Feel free to resolve this and merge.